<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomItem</class>
 <widget class="QDialog" name="RoomItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>156</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #222429;
border-radius: 16px;</string>
  </property>
  <widget class="click_label" name="imageLab">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>156</width>
     <height>156</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 16px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="roomName">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>166</y>
     <width>156</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;微软雅黑&quot;;
font-weight: 500;
font-size: 16px;
color: #FFFFFF;
line-height: 22px;
text-align: left;
font-style: normal;</string>
   </property>
   <property name="text">
    <string>浪漫满屋-仙女避风港</string>
   </property>
  </widget>
  <widget class="QLabel" name="userName">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>194</y>
     <width>156</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: “微软雅黑”;
font-weight: 400;
font-size: 12px;
color: #A8A8A7;
line-height: 17px;
text-align: left;
font-style: normal;</string>
   </property>
   <property name="text">
    <string>房主：▁倔强こ男孩う</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>0</y>
     <width>55</width>
     <height>33</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>55</width>
     <height>28</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #161616;


border-top-left-radius: 0;
    border-top-right-radius: 16px;
    border-bottom-right-radius: 0;
    border-bottom-left-radius: 16px;

opacity: 0.5;</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>8</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="maximumSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../res.qrc">:/images/icon_number.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="hotValue">
      <property name="styleSheet">
       <string notr="true">font-family: &quot;微软雅黑&quot;;
font-weight: 400;
font-size: 14px;
color: #FFFFFF;
line-height: 20px;
text-align: left;
font-style: normal;</string>
      </property>
      <property name="text">
       <string>138</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>click_label</class>
   <extends>QLabel</extends>
   <header location="global">click_label.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
